python/example/berry/berry.py

```diff
@@ -33,26 +33,30 @@ class Berry(inla.INLAModel):
     def __init__(
         self,
         *,
-        p0=np.full(4, 0.1),
-        p1=np.full(4, 0.3),
+        n_arms=4,
```

Comment:
A bunch of changes through this PR are enabling n_arms != 4 for the various Berry estimation methods we've implemented.